andela/ah-baqi-frontend
# #166980206 User signup with email

### What does this PR do?
Adds functionality for user signup
### Description of Task to be completed?
- [x] Form to get user input
- [x] Add state management with redux
- [x] Test for the functionality
### How should this be manually tested?
- Checkout to:

      ft-signup-ui-166980206
     
- Install dependencies:

      yarn
     
- Run local server:

      yarn develop
- Open your browser and navigate to `localhost:3000`
- Attempt to signup
You should be able to sign up and be redirected to a new navbar
### Any background context you want to provide?
Previously lacked this functionality
### What are the relevant pivotal tracker stories?
[#166980206](https://www.pivotaltracker.com/story/show/166980206)
### Screenshots (if appropriate)
<img width="1234" alt="Screen Shot 2019-07-11 at 18 02 54" src="https://user-images.githubusercontent.com/38868860/61062197-6bfccf00-a406-11e9-9aa1-6040e8649b9c.png">
<img width="964" alt="Screen Shot 2019-07-11 at 18 03 00" src="https://user-images.githubusercontent.com/38868860/61062198-6c956580-a406-11e9-8d28-63c02c8e1971.png">
<img width="885" alt="Screen Shot 2019-07-11 at 18 03 10" src="https://user-images.githubusercontent.com/38868860/61062199-6c956580-a406-11e9-927a-5d4b01689725.png">
<img width="889" alt="Screen Shot 2019-07-11 at 18 03 50" src="https://user-images.githubusercontent.com/38868860/61062200-6d2dfc00-a406-11e9-8cb1-e0e2ee783fb9.png">
<img width="1273" alt="Screen Shot 2019-07-11 at 18 04 08" src="https://user-images.githubusercontent.com/38868860/61062201-6d2dfc00-a406-11e9-981c-6d6cd04753ba.png">
